app/src/main/java/com/example/ohthmhyh/User.java

```diff
@@ -4,7 +4,7 @@
 
 public class User {
     private String username, password;
-    private ArrayList<Habit> habitList;
+    public static ArrayList<Habit> habitList;
```

Comment:
Why was this change made?

Reply:
Oh this is me messing around with java and try to make it work. it doesn't work as intended tho. I must have forgotten to remove it when I commit

Reply:
Okay awesome, thanks for letting me know. I will revert the change to this file.